CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+
+## Submission Guidelines
+We use GitHub issues for each Flynt repository to keep track of all incoming queries (feature requests, bug reports, and questions). Please use the most relevant repository for submitting your issue:
+
+- [Flynt Theme Issues](https://github.com/flyntwp/flynt-starter-theme/issues)
+- [Flynt Core Issues](https://github.com/flyntwp/flynt-core/issues)
+- [Flynt Docs Issues](https://github.com/flyntwp/flynt-docs/issues)
+- [Flynt CLI Issues](https://github.com/flyntwp/flynt-cli/issues)
+- [Flynt Generator Issues](https://github.com/flyntwp/generator-flynt/issues)
+- [Flynt Docker Issues](https://github.com/flyntwp/docker-flynt-build/issues)
+- [ACF Field Composer Issues](https://github.com/flyntwp/acf-field-group-composer/issues)
+
+### Issues and Bugs
+Before you submit your issue please search GitHub issues first, maybe your question was already answered.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue. Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues. Providing the following information will increase the chances of your issue being dealt with quickly:
+
+- **Overview of the Issue** - if an error is being thrown a non-minified stack trace helps.
+- **Motivation for or Use Case** - explain why this is a bug for you.
+- **Flynt Version(s)** - is it a regression?
+- **Browsers and Operating System** - is this a problem with all browsers or only specific ones?
+- **Reproduce the Error** - provide a live example, or an unambiguous set of steps.
+- **Related Issues** - has a similar issue been reported before?
+- **Suggest a Fix** - if you can't fix the bug yourself, perhaps you can point to what might be causing the problem (line of code or commit).
+
+## Feature Requests
+You can request a new feature by submitting an issue to the [relevant GitHub Repository](#submission-guidelines). If you would like to implement a new feature then consider what kind of change it is:
+
+- **Major Changes** should be discussed first [in an issue](#issues-and-bugs) so that we can better coordinate our efforts, prevent duplication of work, and help you to craft the change so that it is successfully accepted into the project.
+- **Small Changes** can be crafted and submitted to the [relevant GitHub repository](#submission-guidelines) as a Pull Request.
+
+
+## Pull Requests
+Before you submit a pull request please adhere to the following guidelines:
+
+- Search the [relevant GitHub repository](#submission-guidelines) for an open or closed pull request that relates to your submission. You don't want to duplicate effort.
+
+- Make your changes in a new git branch. **If you are submitting a change for the [Flynt Start Theme](https://github.com/flyntwp/flynt-starter-theme), branch from the `develop` branch, not `master`.**
```

Comment:
Is this actually correct? Can just anyone create a branch on our repos? Don't they have to fork it first and then do a PR from their repo to the main repo? Or am I missing something?